To: vim_dev@googlegroups.com
Subject: Patch 7.3.113
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.113
Problem:    Windows: Fall back directory for creating temp file is wrong.
Solution:   Use "." instead of empty string. (Hong Xu)
Files:	    src/fileio.c


*** ../vim-7.3.112/src/fileio.c	2011-01-17 20:08:03.000000000 +0100
--- src/fileio.c	2011-02-06 13:14:25.000000000 +0100
***************
*** 7459,7465 ****
  
      STRCPY(itmp, "");
      if (GetTempPath(_MAX_PATH, szTempFile) == 0)
! 	szTempFile[0] = NUL;	/* GetTempPath() failed, use current dir */
      strcpy(buf4, "VIM");
      buf4[2] = extra_char;   /* make it "VIa", "VIb", etc. */
      if (GetTempFileName(szTempFile, buf4, 0, itmp) == 0)
--- 7459,7468 ----
  
      STRCPY(itmp, "");
      if (GetTempPath(_MAX_PATH, szTempFile) == 0)
!     {
! 	szTempFile[0] = '.';	/* GetTempPath() failed, use current dir */
! 	szTempFile[1] = NUL;
!     }
      strcpy(buf4, "VIM");
      buf4[2] = extra_char;   /* make it "VIa", "VIb", etc. */
      if (GetTempFileName(szTempFile, buf4, 0, itmp) == 0)
*** ../vim-7.3.112/src/version.c	2011-02-01 21:54:56.000000000 +0100
--- src/version.c	2011-02-09 14:46:12.000000000 +0100
***************
*** 716,717 ****
--- 716,719 ----
  {   /* Add new patch number below this line */
+ /**/
+     113,
  /**/

-- 
'Psychologist' -- Someone who looks at everyone else when
an attractive woman enters the room.

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
